src/LiveComponent/README.md

```diff
@@ -0,0 +1,1158 @@
+# Live Components
+
+**EXPERIMENTAL** This component is currently experimental and is
+likely to change, or even change drastically.
+
+Live components work with the [TwigComponent](../TwigComponent)
+library to give you the power to automatically update your
+Twig components on the frontend as the user interacts with them.
+Inspired by [Livewire](https://laravel-livewire.com/) and
+[Phoenix LiveView](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html).
+
+A real-time product search component might look like this:
+
+```php
+namespace App\Components;
+
+use Symfony\UX\LiveComponent\LiveComponentInterface;
+
+class ProductSearchComponent implements LiveComponentInterface
+{
+    public string $query = '';
+
+    private ProductRepository $productRepository;
+
+    public function __construct(ProductRepository $productRepository)
+    {
+        $this->productRepository = $productRepository;
+    }
+
+    public function getProducts(): array
+    {
+        // example method that returns an array of Products
+        return $this->productRepository->search($this->query);
+    }
+
+    public static function getComponentName(): string
+    {
+        return 'product_search';
+    }
+}
+```
+
+```twig
+<div {{ init_live_component(this) }}>
+    <input
+        type="search"
+        name="query"
+        value="{{ this.query }}"
+        data-action="live#update"
+    >
+
+    <ul>
+        {% for product in this.products %}
+            <li>{{ product.name }}</li>
+        {% endfor %}
+    </ul>
+</div>
+```
+
+As a user types into the box, the component will automatically
+re-render and show the new results!
+
+## Installation
+
+Let's get started! Install the library with:
+
+```
+composer require symfony/ux-live-component
```

Comment:
what about `symfony/ux-twig/component` in reference of the sentence `Live components work with the TwigComponent` ?

Reply:
Hmm, I don't quite understand. Can you say a bit more?

Thanks for the review :)

Reply:
@weaverryan sorry missing the word `adding` :s

=> `what about adding symfony/ux-twig/component` 

so that the line became `composer require symfony/ux-live-component symfony/ux-twig/component`

Reply:
Ah! It's just not needed - since `symfony/ux-twig-component` is a dependency: so you'll get it automatically.